firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigGetParameterHandler.java

```diff
@@ -325,6 +344,35 @@ public FirebaseRemoteConfigValue getValue(String key) {
     return allConfigs;
   }
 
+  /**
+   * Adds a listener that will be called whenever one of the get methods is called.
+   *
+   * @param listener function that takes in the parameter key and the {@link ConfigContainer} JSON
+   */
+  public void addListener(BiConsumer<String, ConfigContainer> listener) {
+    synchronized (listeners) {
+      listeners.add(listener);
+    }
+  }
+
+  /**
+   * Calls all listeners in {@link #listeners}.
+   *
+   * @param key Remote Config parameter key
+   * @param container cache of {@link ConfigContainer}
```

Comment:
The `ConfigContainer` is a local cache of the config, but I'm not sure it makes sense to call it a cache in this context. Maybe `{@link ConfigContainer} containing {@code key}`

Reply:
Done.